fvm/environment/programs.go

```diff
@@ -176,6 +178,45 @@ func (programs *Programs) get(
 	return nil, false
 }
 
+// GetAndSetProgram gets the program from the cache,
+// or loads it (by calling load) if it is not in the cache.
+// When loading a program, this method will be re-entered
+// to load the dependencies of the program.
+//
+// TODO: this function currently just calls GetProgram and SetProgram in pair.
+// This method can be re-written in a far better way by removing the individual
+// GetProgram and SetProgram methods.
+func (programs *Programs) GetAndSetProgram(
```

Comment:
is it too late to rename this to GetOrLoadProgram?

Reply:
cc: @turbolent @SupunS

Reply:
Get or load seems better to me

Reply:
Yeah, we can change it. Could you please open an issue or even PR in the Cadence repo?